internal/database/atendee_table.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+    "database/sql"
+    "fmt"
+    "log"
+
+    _ "github.com/go-sql-driver/mysql"
+)
+
+func CreateAttendeesTable() {
+    // Connect to the database
+    dsn := "root:admin123@tcp(localhost:3306)/events" //dsn
+    db, err := sql.Open("mysql", dsn) //open connection to db
+    if err != nil { //check for errors
+        log.Fatal(err) //exit if error is found
+    }
+    defer db.Close() //close the connection when the function ends
+	
+    // SQL query to create the attendees table
+    createTableQuery := `
+    CREATE TABLE IF NOT EXISTS attendees (
+        id INT AUTO_INCREMENT PRIMARY KEY,
+        event_id INT,
+        first_name VARCHAR(255) NOT NULL,
+        last_name VARCHAR(255) NOT NULL,
+        email VARCHAR(255) NOT NULL,
+        phone_number VARCHAR(20),
+        image_url VARCHAR(255),
+        FOREIGN KEY (event_id) REFERENCES events(id)
```

Comment:
yay foreign keys!! We'll want to remove it in this case because we're removing event_id, but I love the idea of adding it to the RSVP table (for attendee_id when we add that and for event_id as well)!